doc/Publishing.md

```diff
@@ -165,10 +165,50 @@ Format:
     The release will start now. We’ll post an update once it has completed.
     ```
 
-### 2.1.2 OPTION 1: Perform the release LOCALLY
+### 2.1.2 Newly added Theia packages - publish initially to NPM
 <!-- release: both -->
 
-### 2.1.2.1 Prepare the release locally
+_NOTE:_ New `@theia` packages must be published once manually by a Theia committer before the publish workflow can publish them.
+This is due to recent changes requiring trusted workflows for npm publishing.
+
+It is recommend to first publish a next version of the new package, then we can publish the release properly via the recommended publish workflow.
+- To publish locally, you need to:
+  - Ensure you are logged in to NPM (`npm login`; NPM will prompt you for an OTP (one-time password) or security key).
+  - Have your 2FA ready, as you will need an OTP for the publishing process to complete.
+- Run `npm run publish:next`
+  - Optional: If you remove the `--yes` parameter from the publish script, the publishing process will ask you to confirm each step before proceeding.
+
+Once it is published to NPM, please update the settings of this package as follows:
+- Have your security or 2FA ready
+- Go to `https://www.npmjs.com/package/@theia/<new-package>/access`
+- Trusted `Publisher` > Select `GitHub Actions publisher` > Enter the required fields to our publish workflow (`publish-ci.yml`)
+- Set `Publishing access` to `Require two-factor authentication and disallow tokens (recommended)`
+
+Make sure to open a follow-up ticket to notify the team so the package is published initially,  
+and the npm settings are updated to allow trusted workflow publishing before the next release.
+
+### 2.1.3 OPTION 1 (preferred): Perform the release via GH WORKFLOW
+<!-- release: both -->
+
+_NOTE:_ This publishing option is preferred, as the packages are built and signed on GitHub Actions with provenance using the trusted workflow for NPM publishing.
+
+- Run the [_Publish packages to NPM_](https://github.com/eclipse-theia/theia/actions/workflows/publish-ci.yml) workflow
+- Choose the release branch (i.e., `release/{{majorMinor}}.x`)
+- Choose the respective release type and check the input option in case it is a patch for a previous version.
+
+### 2.1.3.1 Check Package update PR
+<!-- release: both -->
+
+- The workflow automatically creates a PR to update the package versions for the release branch, see [example here](https://github.com/eclipse-theia/theia/pull/16438)
+- Follow the instructions in the PR, to ensure all package versions are updated and change the author of the commits to you.
+- Wait for the checks to succeed, then merge using `Rebase and Merge`.
+
+### 2.1.4 OPTION 2: Perform the release LOCALLY
+<!-- release: both -->
+
+_NOTE:_ Performing the release locally will publish no signed packages to NPM.
```

Comment:
```suggestion
_NOTE:_ Performing the release locally will publish unsigned packages to NPM.
```

Or does it infact not publish at all?

Reply:
Thanks! Local publishing should still work, but the packages published to npm will not include the provenance signature.